superset/common/query_object.py

```diff
@@ -72,15 +89,15 @@ def __init__(
             for metric in metrics
         ]
         self.row_limit = row_limit
-        self.filter = filters if filters is not None else []
+        self.filter = filters or []
         self.timeseries_limit = timeseries_limit
         self.timeseries_limit_metric = timeseries_limit_metric
         self.order_desc = order_desc
-        self.extras = extras if extras is not None else {}
-        self.columns = columns if columns is not None else []
-        self.orderby = orderby if orderby is not None else []
+        self.extras = extras or {}
+        self.columns = columns or []
+        self.orderby = orderby or []
 
-    def to_dict(self):
+    def to_dict(self) -> Dict:
```

Comment:
Could we be more explicit on the dict type, i.e., `Dict[str, Any]` (or similar).

Reply:
I kept `Dict` types pretty vague - partly because I wasn't always sure about what type the dict was and partly because most of the Dicts are `[str, Any]` anyways, so this isn't too useful anyways. Still, this example is pretty unambiguous, so I'll change some of the more obvious Dict types like this to be more explicit.

Reply:
Generally I like specificity (I realize that `Dict[str, Any]` is still somewhat vague but it's more specific that thus `Dict`. Note there are probably some dicts which are `Dict[str, str]` (or similar) and thus these type hints could be quite valuable.